packages/component-library/src/components/form/mt-button/mt-button.vue

```diff
@@ -289,6 +289,7 @@ const isInsideTooltip = useIsInsideTooltip();
   width: 100%;
   display: flex;
   justify-content: center;
+  align-items: center;
```

Comment:
Just curious what problem was this solving? Was it that for block buttons they were not centred?

Reply:
In some situation a small block button has not vertical centered text (depending on the outer element, e.g. in the colorpicker as the apply button). This was needed to fix it